core/trino-main/src/main/java/io/trino/operator/output/UnnestingPositionsAppender.java

```diff
@@ -59,12 +69,15 @@ public void appendRle(Block block, int rlePositionCount)
         if (rlePositionCount == 0) {
             return;
         }
```

Comment:
When is `appendRle` called on UnnestingPositionsAppender ?
Can we have nesting of RLE and Dictionary blocks anymore ?